RadarSTATSWidgetExtension/MetricsSummaryWidget.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  MetricsSummaryWidget.swift
+//  RadarSTATSWidgetExtension
+//
+//  Created by Pedro José Pereira Vieito on 2/10/20.
+//
+
+import SwiftUI
+import WidgetKit
+import Intents
+import RadarSTATSIntents
+
+struct MetricsSummaryWidgetProvider: IntentTimelineProvider {
+    func placeholder(in context: Context) -> MetricsSummaryEntry {
+        MetricsSummaryEntry(configuration: GetMetricsSummaryIntent())
```

Comment:
Following the same pattern of a few lines below, I propose the following:
```suggestion
        let configuration = GetMetricsSummaryIntent()
        MetricsSummaryEntry(configuration: configuration)
```